docs/dev-setup.md

```diff
@@ -151,7 +151,8 @@ _Note_: You can run the backend with either `gradle` or with Eclipse. Eclipse do
        - `ANET_DB_DRIVER` : `org.postgresql.Driver`
 
 ### Server side tests
-1. Start with a clean test-database when running tests: `/gradlew -PtestEnv dbDrop dbMigrate dbLoad`
+1. Start with a clean test-database when running tests: `./gradlew -PtestEnv dbDrop dbMigrate dbLoad`
+1. Run your _development_ server with a fake SMTP container: `./gradlew -PtestEnv dockerCreateFakeSmtpServer dockerStartFakeSmtpServer`
```

Comment:
Strictly speaking, with `-PtestEnv` it is the the test server (of your development environment).